### ISSUE TITLE
Which Python versions this package support?

### ISSUE THREAD

**Issue by rfilgueiras** [aqlaboratory/openfold#265]
Python 3.7 is approaching its EOL in 5 months.

Are there plans or blockers to bump the supported Python interpreter to version?

**Comment by gahdritz** [comment]
As far as I know, OF is compatible with newer versions of Python. Did you run into any issues using them?

**Comment by rfilgueiras** [comment]
Thank you @gahdritz for your answer.

I did not start using it, I just checked the project `environment.yaml` and `setup.py` files and found out they only mention Python 3.7.

Which version are you actually using? Are you also upgrading the dependencies mentioned in the `environment.yaml` file?

I was looking for a clear definition of what is support or not but If I understood correctly this is still a grey area.

**Comment by gahdritz** [comment]
Yeah you're right I've been a bit sloppy with versioning. I'm personally still using 3.7 for OF inference, but I've also used newer versions and not run into any issues. I'll double check that everything still works and bump up the Python version in the environment file.

**Comment by vaclavhanzl** [comment]
As 3.7 started to fail in Docker build, I proposed to use 3.9 in #309. Also, 3.9 is what Google Colab uses now and we have to adhere to it in our Colab notebook.
